src/components/Restaurant.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div>
+    <div class="restaurant" v-if="restaurants.length > 0">
+      <div
+        class="restaurant__img"
+        v-if="restaurant.restaurant.featured_image.length > 0"
+        :style="{'background-image': 'url(' + restaurant.restaurant.featured_image + ')' }"
+        alt="restaurant"
+      ></div>
+      <div class="restaurant__img" v-else alt></div>
+      <div class="restaurant__header">
+        <h1>
+          <a :href="restaurant.restaurant.events_url">{{restaurant.restaurant.name}}</a>
+        </h1>
+        <h4>{{restaurant.restaurant.location.locality}}, {{restaurant.restaurant.location.city}}</h4>
+      </div>
+      <div class="restaurant__description">
+        <div class="restaurant__links">
+          <h3>Linki</h3>
+          <a :href="restaurant.restaurant.photos_url">Zdjęcia</a>
+          <br />
+          <a :href="restaurant.restaurant.menu_url">Menu</a>
+          <br />
+          <a :href="restaurant.restaurant.events_url">Wydarzenia</a>
+        </div>
+        <div class="restaurant__info">
+          <strong>Adres:</strong>
+          {{restaurant.restaurant.location.address}}
+          <br />
+          <strong>Kuchnia:</strong>
+          {{restaurant.restaurant.cuisines}}
+          <br />
+          <strong>Średni koszt obiadu dla 2 osób:</strong>
+          {{restaurant.restaurant.average_cost_for_two}}zł
+          <br />
+          <strong>Ocena:</strong>
+          {{restaurant.restaurant.user_rating.aggregate_rating}}
+          <span
+            style="color: rgb(63, 126, 0);"
+          >({{restaurant.restaurant.user_rating.rating_text}})</span>
+          <br />
+          <p
+            class="success"
+            v-if="restaurant.restaurant.has_online_delivery"
+          >Obsługuje przez internet</p>
+          <p class="fail" v-else>Nie obsługuje zamówień przez Internet</p>
+          <p
+            class="success"
+            v-if="restaurant.restaurant.is_table_reservation_supported"
+          >Możliwość rezerwacji miejsc</p>
+          <p v-else class="fail">Brak rezerwacji miejsc</p>
+        </div>
+      </div>
+      <router-link class="restaurant__back" to="/">Back</router-link>
+    </div>
+    <div v-else></div>
+  </div>
+</template>
+
+<script>
+import store from '../store'
+
+export default {
+  computed: {
+    restaurants () {
+      return store.restaurants
+    },
+    restaurant () {
+      if (this.restaurants.length > 0) {
+        return this.restaurants.find(
+          (item) => item.restaurant.id === parseInt(this.$route.params.id)
+        )
+      } else {
+        return {}
```

Comment:
It seems it would work the same as `{ restaurant: {} }`, considering the fact you guard against using that field via a `v-if` anyway. Ergo: the above comment holds.